data_stream_xml_generation/generate_xml_datm.py

```diff
@@ -84,9 +84,12 @@
         "CORE_IAF_JRA55do.LWDN",
         "CORE_IAF_JRA55do.SWDN",
     ]:
-        SubElement(
+        if year_first == year_last:
+            SubElement(stream_info, "offset").text = "0"  #RYF starts at midnight
+        else:    
+            SubElement(
             stream_info, "offset"
-        ).text = "-5400"  # shift back 1.5hr to match RYF
+            ).text = "-5400"  # shift back 1.5hr to match RYF
     else:
         SubElement(stream_info, "offset").text = "0"
```

Comment:
```python
    if (stream_name in [
        "CORE_IAF_JRA55do.PRSN",
        "CORE_IAF_JRA55do.PRRN",
        "CORE_IAF_JRA55do.LWDN",
        "CORE_IAF_JRA55do.SWDN",
    ]) and (year_first != year_last): 
        SubElement(
            stream_info, "offset"
        ).text = "-5400"  # shift back 1.5hr to match RYF
    else:
        SubElement(stream_info, "offset").text = "0"
```

Reply:
Changed in this commit [9f28f27](https://github.com/COSIMA/om3-scripts/pull/17/commits/9f28f27ba493547adf0929f38fc08d2bc22f62b9)